lib/blocks/blockreader.go

```diff
@@ -107,7 +109,7 @@ func (br *Reader) DoTheThing(filename string) error {
 		l := s.Text() + "\n"
 
 		if err := br.LineRead(br, br.LineCount, l); err != nil {
-			return fmt.Errorf("NB LineRead failed @ %s:%d for %s: %v", br.FileName, br.LineCount, l, err)
+			return fmt.Errorf("NB LineRead failed @ %s#%d for %s: %v", br.FileName, br.LineCount, l, err)
```

Comment:
This is reverting a PR to change to :s as such could we revert this change?

Reply:
Oops, indeed! Fixed.

What about https://github.com/katbyte/terrafmt/pull/12/files#diff-9e4bd7d4c75f212c91e8d3bc2283ba3aR134 ?